docs/reference/modules/page.md

```diff
@@ -0,0 +1,379 @@
+      {
+        parkedId: 'teamParkedId',
+        type: 'staff-page',
+        _defaults: {
+          slug: '/team',
+          title: 'Our Team',
+        }
+      }
+    ]
+  },
+  // ...
+}
+```
+
+### `publicApiProjection`
+
+By default the built-in Apostrophe REST APIs are not accessible without proper [authentication](/reference/api/authentication.md). You can set an exception to this for `GET` requests to return specific document properties with the `publicApiProjection` option.
+
+This should be set to an object containing individual field name keys set to `1` for their values. Those fields names included in the `publicApiProjection` object will be returned when the `GET` API requests are made without authentication.
+
+#### Example
+
+```javascript
+// modules/@apostrophecms/page/index.js
+module.exports = {
+  options: {
+    publicApiProjection: {
+      title: 1,
+      _url: 1 // 👈 Dynamic properties are allowed
+    }
+  },
+  // ...
+}
+```
+
+Unauthenticated [`GET /api/v1/@apostrophecms/page`](/reference/api/pages.md#get-api-v1-apostrophecms-page) requests would return each piece with only the `title` and `_url` properties.
+
+### `quickCreate`
+
+Pages are included in the admin bar "quick create" menu by default. Setting `quickCreate: false` on the page module will disable this.
+
+#### Example
+
+```javascript
+// modules/@apostrophecms/page/index.js
+module.exports = {
+  options: {
+    quickCreate: false
+  },
+  // ...
+}
+```
+
+### `types`
+
+The `types` array defines the page types available to users when creating or editing pages. Each item in the array should have a `label` property and a `name` property, which matches an active page type. If no `types` array is set, only the core "Home" page type will be available.
+
+[Parked pages](#park) may use page types that are not in the `types` option array. This allows developers to do things such as parking a single search page but not allowing users to create additional search pages.
+
+#### Example
+
+```javascript
+// modules/@apostrophecms/page/index.js
+module.exports = {
+  options: {
+    types: [
+      {
+        name: 'default-page',
+        label: 'Default'
+      },
+      {
+        name: 'article-page',
+        label: 'Article Index'
+      }
+      {
+        name: '@apostrophecms/home-page',
+        label: 'Home'
+      }
+    ]
+  },
+  // ...
+}
+```
+
+## Related documentation
+
+- [Pages guide](/guide/pages.md)
+- [Pages REST API](/docs/reference/api/pages.md)
+
+## Featured methods
+
+The following methods belong to this module and may be useful in project-level code. See the [source code](https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/page/index.js) for all methods that belong to this module.
+<!-- Some are used within the module and would just create noise here. -->
+
+Because this module has an alias, you can call these from another module from the alias path. For example, `self.apos.page.find()`.
```

Comment:
I would just say "using the alias." "Alias path" sounds like aliases are something more complicated than they are (just names).